docs/developer/checkout.mdx

```diff
@@ -1385,6 +1385,18 @@ A response would look like this:
 }
 ```
 
+:::caution
+When any error will be raised during the checkout completion or the payment
+will become inactive, processed payment will be refunded or voided (in case the payment wasn't charged yet).
```

Comment:
```suggestion
will become inactive, processed payment will be refunded or voided (in case it still needs to be charged).
```

Reply:
I'm not sure if it's gonna be more clear. For me, it will suggest that this payment will still need to be charged.